examples/values-ez.yaml

```diff
@@ -131,8 +226,8 @@ scaffold:
             path: /
     deployment:
       registry: quay.io
-      repository: ablock/tuf-server
-      version: "sha256:c4f534807b331e97b107d908b7e0cf14a896399bc372bca6a2c7e4f6d730807c"
+      repository: securesign/tuf/server
+      version: latest
```

Comment:
Probably we should be using specific shas here instead of floating tags - @sallyom wdyt?

Reply:
This is actually something @sallyom mentioned to me, this one just slipped through the cracks, ill make the change there now :+1:

Reply:
great! I mentioned using a tag that is not `latest` but what @lance suggested - the sha256- is better than pinning a tag (like 1.2.0)